CHANGELOG.md

```diff
@@ -13,7 +13,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * avifenc, avifdec: Allow "-j all" to automatically use all of the cores on the machine (#670)
 
 ### Changed
-
+* Re-map libavif speed 7-10 to libaom speed 7-9 (#682)
+*  Refer to https://aomedia-review.googlesource.com/c/aom/+/140624
+*  If you were using libaom with lower encoder ABI version
```

Comment:
Please remove "with lower encoder ABI version". The reason is that we change the mapping of speeds 7-10 whether or not all intra mode is used.

Reply:
Done